extensions/smallrye-opentracing/runtime/src/main/java/io/quarkus/smallrye/opentracing/runtime/TracerProducer.java

```diff
@@ -30,6 +31,7 @@
 public class TracerProducer {
     @Produces
     @Dependent
+    @ApplicationScoped
```

Comment:
Wouldn't `@Dependent` need to be removed?

If moving to `@ApplicationScoped` not sure you'd get an `InjectionPoint` injected to the method

Reply:
> Wouldn't @Dependent need to be removed?

I don't know maybe @mkouba ?

> If moving to @ApplicationScoped not sure you'd get an InjectionPoint injected to the method

We don't seem to be using it.

Reply:
Yes, if this does not fail the build it is a bug in Arc's validation. In this case though if we don't actually need the InjectionPoint it can be removed.

Reply:
`InjectionPoint` and non-dependent scope is a known issue: https://github.com/quarkusio/quarkus/issues/2131

Reply:
And of course, having two scopes declared is illegal and should result in deployment error. I thought we have an issue for this too but I could not find it so I've created a new one: https://github.com/quarkusio/quarkus/issues/2178

Reply:
I have removed the `@Dependent`  and injection point.